packages/package/src/typescript.js

```diff
@@ -74,7 +74,7 @@ export async function transpile_ts(filename, source) {
 		compilerOptions: {
 			...options,
 			module: ts.ModuleKind.ESNext,
-			moduleResolution: ts.ModuleResolutionKind.NodeJs // switch this to bundler in the next major, although it probably doesn't make a difference
+			moduleResolution: ts.ModuleResolutionKind.NodeNext
```

Comment:
I believe setting `moduleResolution` to `NodeNext` also requires setting `module` to `NodeNext` as well!

![img](https://i.imgur.com/O4dirod.png)

Reply:
yes! thanks! done